slaves/linux-cross/arm-linux-gnueabihf.config

```diff
@@ -0,0 +1,577 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Crosstool-NG Configuration
+#
+# Generated with crosstool-ng-1.22.0's `ct-ng menuconfig` command using the default configuration
+# plus the following modifications
+#
+# Target options > Target Architecture = arm
+# Target options > Architecture level = armv7-a -- (*)
```

Comment:
Sure we can build these two toolchains, one for each target.

My only worry about this is that the `armv7-unknown-linux-gnueabihf` uses the `arm-linux-gnueabihf-gcc` by default so we'll need to override the gcc used by the build system. I don't know how to do that but I hope you do!